src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/Calendar.java

```diff
@@ -31,12 +31,12 @@ public Calendar() {
   }
 
 
-  public String getId() {
-    return id;
+  public String getCalendar_id() {
+    return calendar_id;
   }
 
-  public void setId(String id) {
-    this.id = id;
+  public void setCalendar_id(String calendar_id) {
+    this.calendar_id = calendar_id;
   }
```

Comment:
Please follow the styleguide, variables should be always lowerCamelCase. Please correct.